features/step_definitions/check_your_answers_steps.rb

```diff
@@ -39,7 +70,11 @@
 end
 
 And("I am viewing the means summary check your answers page") do
```

Comment:
Is this the means summary or the means capital check your answers page now?

Reply:
Yeah, should now be capital CYA page, will update